ci/install.sh

```diff
@@ -14,12 +14,19 @@ apt-get install -y \
   postgresql-server-dev-{10,11,12,13,14} \
   postgresql-{10,11,12,13,14}-postgis-3 \
   postgresql-{10,11,12,13,14}-pgextwlist
+
+# Install aiven-extras.
 git clone https://github.com/aiven/aiven-extras /aiven-extras
 git -C /aiven-extras checkout bf259febf0fd1e76479d4d4f252cc36a4ff9767f
 make -C /aiven-extras clean \
   aiven_extras.control \
   sql/aiven_extras--1.1.8.sql \
   sql/aiven_extras--1.1.7--1.1.8.sql
+
+# Clone aiven-pg-security (aiven_gatekeeper): https://github.com/aiven/aiven-pg-security/releases/tag/v1.0.4
+git clone https://github.com/aiven/aiven-pg-security/ /aiven-pg-security
+git -C /aiven-pg-security checkout 3107f450ee765515a8a19c78e4f3125ce6ab1f36
```

Comment:
This one is obsolete, I'd rather checkout main or latest tag (which is v1.0.6 - Stable), which should be nice to have somewhere explicitly (top of shell script as var?) 
Also probably would be nice to bump aiven-extras as well.

Reply:
Great idea. Have a look at the new commit I pushed.

- `aiven-extras`: the latest tag is used, because there are no GitHub Releases on this repository.
- `aiven-pg-security`: the tag from the latest release is used. Newer pre-releases are ignored.